.github/workflows/e2e_android.yml

```diff
@@ -106,7 +106,7 @@ jobs:
           force-avd-creation: false
           disable-animations: true
           arch: x86_64
-          avd-name: Pixel_4_XL_API_31
+          avd-name: Pixel_4_XL_API_28
```

Comment:
This is in a local JSON file so we should just be able to pull it in so there's no need to keep it in sync https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/.detoxrc.json#L28

Reply:
added

Reply:
after playing with this for a while I was able to extra the name from the json file, but then this block of code no longer runs with this error:

Creating AVD.
  /bin/sh -c \echo no | avdmanager create avd --force -n Pixel_4_XL_API_28 --abi 'default/x86_64' --package 'system-images;android-28;default;x86_64' --device 'pixel_4_xl'
  Error: Package path (-k) not specified. Valid system image paths are:
  system-images;android-28;default;x86_64
  null
  
  So reverting back to the hard coded way that works as I tried trimming white space and everything else that might be going on. The command from what I can tell in the logs is the exact same and hard coded works and fetching from the file does not